ui_framework/src/test/find_test_subject.js

```diff
@@ -0,0 +1,10 @@
+// Extract the DOM node which matches a specific test subject selector.
+export const findTestSubject = (mountedComponent, testSubjectSelector, isDOMNode = true) => {
```

Comment:
Thinking a bit more on this. Personally I also find boolean args difficult to read. I think I'd personally just move the `getDomNode` outside this function. I feel it's easier to read `findTestSubject(myComponent, 'foo').getDOMNode()`. We also just needed `getDOMNode` 3 times before, so it doesn't feel to bad to do that (but maybe we need it more now because of changes in enzyme?)

Reply:
I really like your point about boolean args -- I have the same difficulty about them. I also like your suggestions. We'll try it out.